### PR TITLE
new gatsby starter

### PR DESCRIPTION
**Pull request by stamigos**
https://trello.com/c/HkHLgOl3/449-create-a-new-gatsby-starter

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -31,19 +31,9 @@ module.exports = {
       },
     },
     {
-      resolve: `gatsby-source-buttercms`,
+      resolve: require.resolve(`gatsby-source-buttercms`),
```
Comment (benrobertsonio) [inline gatsby-config.js:34]: Is there a reason this uses a local version of the source plugin, rather than installing from npm?

Comment (benrobertsonio) [reply gatsby-config.js:34]: seems like using a local version will prevent people using the starter from opting in to updates to the plugin in the future

Comment (stamigos) [reply gatsby-config.js]: Thanks for the feedback! It totally makes sense. However this one: `require.resolve('gatsby-source-buttercms')` was also taking plugin from `node_modules`. Anyway, using `require.resolve()` isn't necessary here.

Comment (stamigos) [reply gatsby-config.js]: Fixed now ✅

Comment (benrobertsonio) [reply gatsby-config.js]: gotcha!

**Pull request merged**